src/mt2_Lallyver2.h

```diff
@@ -533,7 +533,7 @@ double NewDeltaFinder(double l_delta0, double l_delta, int bisectDivisor, int bi
                 {
                     //Need to transpose function candidate UB to check there is only one root below the candidate UB value and above the candidate LB (awkward check but couldn't think of more efficient way)
                     const double checkdeltasq = checkdelta * checkdelta;
-                    const double checkdeltasqsq = checkdeltasq * checkdeltasq;
+                    // const double checkdeltasqsq = checkdeltasq * checkdeltasq;

```

Comment:
Can you revert changes in this file please? It's a reference implementation from a third party, so we don't want to change it (at least not in-place).
Recommended implementation for now is `mt2_bisect.h`

Thanks!